lib/generators/sufia/templates/workflow.json.erb

```diff
@@ -9,19 +9,45 @@
                     "name": "deposit",
                     "from_states": [],
```

Comment:
My understanding is that the first action without a state becomes the "deposit" action.. https://github.com/projecthydra/curation_concerns/blob/master/app/services/curation_concerns/workflow/workflow_factory.rb#L54

So I don't think we have a `from_state` here and just have other actions transition to `pending_review` as a sort of state that admin/reviewers interact with for mediating works?

We briefly discussed that there would need to be some mechanism that sets a default role for all registered users.. Definitely interested in some ideas on how to best wire this up.